CHANGELOG.md

```diff
@@ -12,6 +12,22 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
+* None.  
+
+## 1.8.0.beta.2
```

Comment:
no need for this its get added during release. Just put it under `master` bug fixes for now.